src/script/view_model/content/MessageListViewModel.ts

```diff
@@ -330,7 +330,7 @@ export class MessageListViewModel {
     if (lastMessage) {
       if (!this.isLastReceivedMessage(lastMessage, this.conversation())) {
         // if the last loaded message is not the last of the conversation, we load the subsequent messages
-        return this.conversationRepository.getSubsequentMessages(this.conversation(), lastMessage, false);
```

Comment:
@AndyLnd are you sure removing this return statement is safe?

Reply:
I re-added the `return`.